ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java

```diff
@@ -5611,4 +5365,282 @@ private enum TableType {
     JDBC
   }
 
+  class OrderByRelBuilder {
```

Comment:
This class is only used in `CalcitePlanner` - probably we can make it `private`? Also some of its methods are missing access modifiers.

Reply:
I made the class private. Unfortunately this is a non-static inner class. It would be better to make it static or move it to a new file but it references methods like `CalcitePlannerAction.genAllRexNode`.
So the methods and the fields are accessible from the `CalcitePlannerAction` instance even if they are declared as private.
I tried to set methods and fields as `private` which should not be accessed outside the class and make `package private` the methods which should be used.